<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="rviz" default="true"/>
    <rosparam command="load" file="$(find fabrics_bridge)/config/fabrics_pointrobot_config.yaml"/>

    <group ns="fabrics">
        <!-- start fabric node -->
        <node pkg="fabrics_bridge" type="pointrobot_fabrics_node" name="pointrobot_fabrics_node" output="screen"/>
    </group>
    <!-- <node pkg="rviz" type="rviz" name="rviz" args="-d $(find fabrics_bridge)/config/fabrics_boxer_visualization.rviz" if="$(arg rviz)"/> -->

</launch>
